app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split(" ")
+    index_new_directories = command_list[2].rfind("/")
```

Comment:
Got a problem here - you're getting last index, where "/" is found. But when there's no "/" it returns -1.
For example, 'mv file.txt file2.txt', there's no /, so index is -1